packages/apps-config/src/api/spec/uart.ts

```diff
@@ -21,17 +21,20 @@ const definitions: OverrideBundleDefinition = {
         Value: 'Vec<u8>',
         ReasonIndex: 'u32',
         LottoIndex: 'u32',
+        MemberId: 'u64',
+        ProposalId: 'u64',
+        TokenBalance: 'Balance',
         LotteryKind: {
-          _enum: {
-            Routine: 'Null',
-            TreasuryFunded: 'ReasonIndex'
-          }
+          _enum: [
+            'Routine',
+            'TreasuryFunded(ReasonIndex)'
```

Comment:
The enums are still incorrect, the correct form is indeed as it was in the original.

```js
Routine: 'Null', // this one is has not extra data
TreasuryFunded: 'ReasonIndex' // This one contains data
```

As mentioned originally, this affects the `LotteryKind` (original correct, new version is not), `LotteryResult` (same where original is correct, new version is incorrect as will match as a c-like enum)